### ISSUE TITLE
Improvement: Remove the max. pod limit for toxic rain

### ISSUE THREAD

**Issue by Ydoum** [PathOfBuildingCommunity/PathOfBuilding#4305]
### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Is your feature request related to a problem?

Currently the amount of maximum pods is set to be equivalent to the max. amount of arrows fired. This, however, ignores your attack speed or any other potential checks that one would like to make.

### Describe the solution you'd like

Remove the limitation, add the base value of "if not maximum" based around the current calculations, and allow us to change the amount of pods freely.

In other words, this should work similarly to Storm Rain's "overlapping beams" tab.

### Describe alternatives you've considered

n/a

### Additional context

My example:

Projectlike count: 11
Max pods: 11

However:

Attack speed: 5.1
The gluttonous tide: extra 9 projectiles that are not calculated

As a result, I should be able to set the amount of pods to 100 (20 proj. x 5 attacks per second).

**Comment by deathbeam** [comment]
This is not how TR works or even how the config works tho. Attack speed is already accounted into DPS so the config optiion is basically pods per attack. Technically this should not even exist anymore and should just be full dps counter setting that you do. Also TR was nerfed and I dont think its possible to overlap more than 3 or 4 pods anymore anyway so dont bother with stacking that many projectiles anymore rly.

**Comment by QuickStick123** [comment]
Closing as invalid as the current max for pods is the number of arrows you fire which is the number number of pods and hence you cannot overlap more than you fire.
